internal/ast/ast.go

```diff
@@ -184,9 +186,9 @@ const (
 	ScopeLocal
 )
 
-// Variable reference (special var, global, or local). Index is the
-// resolved variable index used by the interpreter; Name is the
-// original name used by String().
+// VarExpr is a variable reference (special var, global, or local).
+// Index is the resolved variable index used by the interpreter; Name
```

Comment:
I looked and figure based on common lengths that we are using max 70 columns here.